package.json

```diff
@@ -22,7 +22,7 @@
   "license": "MIT",
   "devDependencies": {
     "broccoli-asset-rev": "^2.5.0",
```

Comment:
Change to `^2.4.5` to be consistent with what `ember-cli` is currently at and with the other repos doing the same.